### ISSUE TITLE
A11y link color in text block

### ISSUE THREAD

**Issue by oleksis** [BlackPythonDevs/blackpythondevs.github.io#156]
Add to the [`bpd.css`](https://github.com/BlackPythonDevs/blackpythondevs.github.io/blob/64a13a03c949710d1c9f23fa54585a23a4cae485/assets/css/bpd.css#L358)

```css
a {
    color: #4c77a6;
    text-decoration: none;
}
```
